### ISSUE TITLE
Minor issue with BLAST

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1512]
Can we please alphabetize the choices?  This is really hard to parse:

![image](https://cloud.githubusercontent.com/assets/7661533/26007323/a1817498-3705-11e7-80fb-cef5e2a0ca26.png)

**Comment by hyoo** [comment]
This is a genome selector issue.

**Comment by hyoo** [comment]
I think genome selector fixed this issue.

![image](https://user-images.githubusercontent.com/1236467/30504153-c3225d7a-9a32-11e7-882d-818ad38fa351.png)
